README.md

```diff
@@ -57,12 +57,16 @@ services:
     image: postgres:14
     container_name: postgres_setup
     restart: on-failure
+    depends_on:
+      - postgres
```

Comment:
`depends_on` does not wait for PostgreSQL to be ready and have a port open. From https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on:

> depends_on does not wait for db and redis to be “ready” before starting web - only until they have been started. If you need to wait for a service to be ready, see Controlling startup order for more on this problem and strategies for solving it.

So we that change does not fix the problem in general, only on some hosts with particular timings. For example, it does not change anything on my development laptop.
And also there is no problem :) Thanks to `restart: on-failure` line, we just restart the setup container until it finishes successfully.